src/Page.js

```diff
@@ -7,6 +7,7 @@ export class Page extends Component {
 	constructor(props) {
 		super(props);
 		this._node = null;
+		this.getPageStyles = this.getPageStyles.bind(this);

```

Comment:
We don't need to use `bind` here because `getPageStyles` isn't being passed into another class.